cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCards.kt

```diff
@@ -0,0 +1,258 @@
+package com.decathlon.vitamin.compose.cards
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithCache
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.buttons.VitaminButtons
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.decathlon.vitamin.compose.foundation.vtmnLightColorPalette
+
+object VitaminCards {
+    /**
+     * A card with image at top
+     * @param painter the [Painter] resource for the image
+     * @param contentDescription to describe image for screen reader. Set to null only if image is decorative.
+     * @param content The content of the card. Can be a composable from [TopImageContent] or a custom composable.
+     * @param modifier The `Modifier` to be applied to the all card.
+     * @param shape to change the shape of the card.
+     * @param imageScale The scale [VitaminCardImageScale] of the image in the card. can be
+     * [VitaminCardImageScale.Fit] or
+     * [VitaminCardImageScale.Full]
+     * @param imageOverlayAlignment The overlay position if overlay is set. Default is [Alignment.TopStart]
+     * @param imageOverlay The overlay to set upon the image. No overlay by default.
+     */
+    @Composable
+    fun TopImage(
+        painter: Painter,
```

Comment:
During the review, I think again about this comment. I don't know if it should be an api slot because the image seems the only mandatory for almost all variants of the card but we should definitely add all parameters useful for an image to be sure that the consumer can customize it.